data/competencies.yml

```diff
@@ -889,18 +901,6 @@
   area: leadership
   domain: null
 
-- id: de580f8a-633d-43bb-b970-9b2987e07b86
-  summary: Actively fosters an inclusive team culture
-  examples:
-    - Celebrates good work publicly and encourages the team to do the same
-    - Spots problems between team members and helps to resolve them
-    - Models inclusive behaviour to the rest of the team
```

Comment:
As we are getting used to working remotely, meetings are going to be an unescapable part of our way of working. Maybe we should have something around that too - 

- Is an ally for minority groups and facilitates discussions in a way that grants an opportunity for everyone present to provide their thoughts. 
- Reinforces the rule that there are no stupid questions and bad ideas.

When I first joined the FT, the latter point gave me a lot of confidence to share my thoughts openly and I think we should definitely encourage more of that !

Reply:
"Reinforces the rule that there are no stupid questions and bad ideas." I really like that

Reply:
I like the "makes space for everyone to be heard"!

Reply:
I like these examples Nikita, we do have a full competency for "Facilitates productive discussions with clear outcomes", I wonder whether these might apply to this instead/as well? I'm not sure what a good split is.

Some possible steps here are:

- We take the second half of your first example "... facilitates discussions in a way that grants an opportunity for everyone present to provide their thoughts." and make this explicit in the ["facilitates discussions" competency](https://github.com/Financial-Times/engineering-progression/blob/master/data/competencies.yml#L547). This can happen as a separate PR I think

- I'd really like to include the first part of that example "is an ally for minority groups" in this competency as suggested, do we think this is measurable enough as is? Possibly it's fine if we also include add a supporting URL which outlines what good allyship looks like

- "Reinforces the rule that there are no stupid questions and bad ideas" I really like this. Possibly also comes under the "facilitates discussions" competency 🤔 what are your thoughts?